Source/santasyncservice/SNTPolaris.h

```diff
@@ -0,0 +1,21 @@
+/// Copyright 2025 North Pole Security, Inc. All rights reserved.
```

Comment:
nit: Should use the updated blob format: https://www.apache.org/licenses/LICENSE-2.0#apply